source/iterators.tex

```diff
@@ -5088,14 +5115,24 @@
 
 \indexlibrarymember{base}{counted_iterator}%
 \begin{itemdecl}
-constexpr I base() const;
+constexpr I base() const & requires CopyConstructible<I>;
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
 \effects Equivalent to: \tcode{return current;}
 \end{itemdescr}
 
+\indexlibrarymember{base}{counted_iterator}%
+\begin{itemdecl}
+constexpr I base() &&;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns \tcode{std::move(current)}.
```

Comment:
Bah, the ingress paper had incorrect context for line 5123 so we didn't notice the "*Effects*: Equivalent to:  `return current;`" / "*Returns*: `std::move(current)`." inconsistency.  I don't suppose I could convince the Editors to reword this paragraph to "*Effects*: Equivalent to: `return std::move(current);`" for consistency to save the trouble of an LWG issue?

Reply:
I'm afraid that's a normative change, but it's clear that that was what was intended.  @zygoloid what do you think?

Reply:
@CaseyCarter Please open a separate issue for this (might have to be a LWG issue); I'm not comfortable making that change as part of this motion.  Thanks!